scripts/BenchmarkMaliciousIP.java

```diff
@@ -0,0 +1,79 @@
+package com.google.netpcapanalysis.lookup_ip;
+
+import com.google.netpcapanalysis.dao.MaliciousIPDaoImpl;
+import java.util.ArrayList;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.BufferedReader;
+import com.google.netpcapanalysis.mockdata.MockDataLoader;
+import com.google.netpcapanalysis.models.PCAPdata;
+import com.google.netpcapanalysis.interfaces.dao.MaliciousIPDao;
+
+
+public class BenchmarkMaliciousIP {
+  private String CSV_FILE = "maliciousIPTest.csv"; //first 1-30 valid IP's, 31-60 malicious IP's
+  ArrayList<PCAPdata> IPlist;
+    long AvgTime;
```

Comment:
variable naming is always Camel Case except for constants. And no underscores. Its ok for this PR but try to follow the convention next time.